public/js/post/update_post.js

```diff
@@ -214,7 +214,7 @@ function checkPostCompleted() {
 function updateNavPost() {
     'use strict';
 
-    var navPost = $('#nav-links')[0].getElementsByTagName('li')[3].getElementsByTagName('a')[0];
+    var navPost = $('#nav-links')[0].getElementsByTagName('li')[4].getElementsByTagName('a')[0];
```

Comment:
Yeah, I definitely think so. But in any case, right now this had been removed since it was decided we won't be updating the nav bar with the POSt credit count at the moment.